### ISSUE TITLE
Lifelong learning and digital literacy

### ISSUE THREAD

**Issue by SvdKraak** [mozilla/global-sprint#287]
**[ Project Contact ]** @svdkraak
**[ GitHub Repo ]** https://github.com/SvdKraak/Lifelong-learning-and-digital-literacy
**[ Track ]** Web Literacy, 
**[ Location ]** The Netherlands, CET
**[ Coach ]** Your OL5 Mentor

### Description
Increasing digital literacy of youth workers' will support youth workers in their lifelong learning attitude and will increase digital competenties and habits, so youth workers will connect easier online with young people and can issue open badges to the young people they work with.
***

## Want to contribute to this project during #mozsprint?
Join us at the Global Sprint, May 10-11. Leave a comment below if you're interested in contributing to this project during #mozsprint 2018!

**Comment by abbycabs** [comment]
Hey @SvdKraak 

Thanks for submitting your project to #mozsprint!

You're almost done your registration. Please complete the first 6 items in the checklist 📝 
If these aren't completed by May 9th, we'll be closing this issue. Let us know if you have any questions! Happy Sprinting 🎉

**Comment by abbycabs** [comment]
Hi @SvdKraak please add the `mozsprint` label to some of your issues. The Sprint has started!

**Comment by codefusser** [comment]
I'm an IT Generalist, have strong passion for youths and would like to be a part of this project. #mozsprint

**Comment by SvdKraak** [comment]
Great! Thank you @codefusser. Unfortunately, I don't understand Github very well. Please forgive me when I do something silly ;-). In which way would you like to be part of it?

**Comment by codefusser** [comment]
If you need resources to guide the youths on how to best maximize the internet for their all round benefits. Perhaps, a researcher, designing a website or in any way other I could support using my skills.

**Comment by SvdKraak** [comment]
I'm searching for ideas how to teach in a non-formal and fun way 'problem solving' competences. We use https://ec.europa.eu/jrc/en/publication/eur-scientific-and-technical-research-reports/digcomp-21-digital-competence-framework-citizens-eight-proficiency-levels-and-examples-use as competences model. One of the competences is problem solving. Any ideas how to teach them with using IT? We are thinking about an online game or something like that. Also realising that often youth workers want to work offline and in real connection with youth.

**Comment by codefusser** [comment]
That's interesting! I've experience in the teaching profession and believe there's always a way to which IT could be used to aid teaching methodologies, most effectively. What kind of audience would you be addressing, for instance age range?

**Comment by codefusser** [comment]
Couldn't download that link in ".pdf" format?

**Comment by SvdKraak** [comment]
We are developing 2 projects: 1. for young people age 16 til 24 with distance to the labour market and we are developing a training course for them on digital competences (outside school learning). 2. for youth workers (professional or volunteer with no age limit), we have created several Open Badges systems to acknowledge the learning outside schools for young people with fewer opportunities. When youth workers need to validate evidence from quests within these Open Badges, we realised that often they don't fully understand how to do this, how to support the yp. Most of these youth workers are above 35 years old.  We also have a group of people who don't see the benefits.
